### PR TITLE
Add prometheus exporter to the Analytics bundle

### PR DESCRIPTION
**Pull request by ClementGautier**
# Description

Prometheus is a time-series data store. We use it for the cloud for monitoring dashboard and also for clients dashboard (so they can know if they are reaching their quota).

# Definition Of Done

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | :ok: 
| Added integration tests           | -
| Changelog updated                 | :ok: 
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | :ok: 
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -116,6 +116,7 @@ protected function getPimDependenciesBundles()
             new Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle(),
             new FOS\OAuthServerBundle\FOSOAuthServerBundle(),
             new Oneup\FlysystemBundle\OneupFlysystemBundle(),
+            new TweedeGolf\PrometheusBundle\TweedeGolfPrometheusBundle(),
```
Comment (nidup) [inline app/AppKernel.php]: not possible in a patch

File: composer.json [hunk 2]
```diff
@@ -70,7 +70,8 @@
         "symfony/symfony": "3.3.6",
         "symfony/polyfill-apcu": "1.4.0",
         "twig/extensions": "1.2.0",
-        "box/spout": "2.7.2"
+        "box/spout": "2.7.2",
+        "tweedegolf/prometheus-bundle": "^0.2.2"
```
Comment (nidup) [inline composer.json]: not possible in a patch

File: app/AppKernel.php (hunk 1 above)
Comment (ClementGautier) [reply app/AppKernel.php]: base changed to master

File: composer.json (hunk 2 above)
Comment (ClementGautier) [reply composer.json]: base changed to master

**Comment by ClementGautier** [comment]
@skeleton base changed to master

File: app/AppKernel.php (hunk 1 above)
Comment (BitOne) [reply app/AppKernel.php]: I don't think it's a good idea to add this kind of very specific bundle to the standard configuration of the PIM.

File: composer.json (hunk 2 above)
Comment (BitOne) [reply composer.json]: Why not adding this bundle only for the cloud installations ? Shouldn't be in the standard base.

**Pull request closed without merging**